Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.39914&attempt_id=9ae6da7c-142d-46b4-b467-3683cff4365a)

Attempted: gcc49

<details><summary>Partial log (click to expand)</summary><p>

```
rm gcc.pod
make[3]: Leaving directory '/private/tmp/nix-build-gcc-4.9.4.drv-0/build/gcc'
make[2]: *** [Makefile:4235: all-stage1-gcc] Error 2
make[2]: Leaving directory '/private/tmp/nix-build-gcc-4.9.4.drv-0/build'
make[1]: *** [Makefile:17844: stage1-bubble] Error 2
make[1]: Leaving directory '/private/tmp/nix-build-gcc-4.9.4.drv-0/build'
make: *** [Makefile:18149: bootstrap] Error 2
builder for '/nix/store/4yp2md3wqw36zxwad8x543vc01wqj0iz-gcc-4.9.4.drv' failed with exit code 2
cannot build derivation '/nix/store/cyw66czqaiymbl5r6g26f7g79xkdvqnj-gcc-wrapper-4.9.4.drv': 1 dependencies couldn't be built
error: build of '/nix/store/cyw66czqaiymbl5r6g26f7g79xkdvqnj-gcc-wrapper-4.9.4.drv' failed
```
</p></details>

--- a/pkgs/development/compilers/gcc/4.9/default.nix
+++ b/pkgs/development/compilers/gcc/4.9/default.nix
@@ -496,8 +496,7 @@ stdenv.mkDerivation ({
     platforms =
       stdenv.lib.platforms.linux ++
       stdenv.lib.platforms.freebsd ++
-      stdenv.lib.platforms.illumos ++
-      stdenv.lib.platforms.darwin;
+      stdenv.lib.platforms.illumos;
   };
 }
 

